builder/files/bor.service

```diff
@@ -6,11 +6,39 @@
 [Service]
   Restart=on-failure
   RestartSec=5s
-  WorkingDirectory=$NODE_DIR
-  EnvironmentFile=/etc/matic/metadata
-  ExecStart=/usr/local/bin/bor $VALIDATOR_ADDRESS
+  ExecStart=/usr/local/bin/bor \
+    --datadir /var/lib/bor/data \
+    --port 30303 \
+    --http --http.addr '0.0.0.0' \
+    --http.vhosts '*' \
+    --http.corsdomain '*' \
+    --http.port 8545 \
+    --ipcpath /var/lib/bor/bor.ipc \
+    --http.api 'eth,net,web3,txpool,bor' \
+    --syncmode 'full' \
+    --networkid '80001' \
```

Comment:
This is for mumbai, how do we specify mainnet networkid 137?

Reply:
Gotcha, probably as you said it's better to use `--bor-mainnet` and `--bor-mumbai`